docs/source/contributing/development-environment.rst

```diff
@@ -60,13 +60,13 @@ Substitute ``cargo build`` with ``cargo test`` to test, or ``cargo check`` to ch
 
 .. note::
 
-  If linking fails when runnning ``cargo test``, you may need to build without the ``pyo3/extension-module`` feature (`context here <https://pyo3.rs/v0.25.1/faq.html#i-cant-run-cargo-test-or-i-cant-build-in-a-cargo-workspace-im-having-linker-issues-like-symbol-not-found-or-undefined-reference-to-_pyexc_systemerror>`_):
+  If linking fails when runnning ``cargo test``, you may need to build without the ``extension-module`` feature (`context here <https://pyo3.rs/v0.25.1/faq.html#i-cant-run-cargo-test-or-i-cant-build-in-a-cargo-workspace-im-having-linker-issues-like-symbol-not-found-or-undefined-reference-to-_pyexc_systemerror>`_):
 
   .. code-block:: bash
 
-    cargo test --features=untrusted,polars,ffi,derive
+    cargo test --no-default-features --features=partials,use-openssl,untrusted,polars,ffi,derive
```

Comment:
If it's important to keep this in sync, would it make any sense to pull this into a shell script, and reference the script here and in CI?

Reply:
I feel like "test with all features but extension-module" is clearer than "test via some other shell script", even if it's a bit longer. I can see the argument for the script though.